doc-botonic/docs/plugins/21.botonic-plugins.md

```diff
@@ -3,9 +3,13 @@ id: botonic-plugins
 title: Introduction to Botonic Plugins
 ---
 
-Sometimes you may need a specific feature according to your needs and you want to build
-your particular plugin that doesn't exist yet. In order to achieve this, we have to review some important aspects.
+Botonic plugins allow you to add more functionalities to your bot. You can:
```

Comment:
st like:
Botonic plugins allow implementing independently functionalities which can be injected into any bot. Once integrated, a plugin is able to:
* Read and update any user input sent to the bot. This allows for instance to inject a NU plugin which tries to guess the intent of the user from their textual input.
* Read and update any output provided by the bot to a user input. This allows for instance to postprocess the bot output, or to track the bot behaviour to an external database or service.